ui/src/containers/PlatformFilter/PlatformFilter.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { FakeHub } from '../../api/testutil';
+import { createProviderAndStore } from '../../store/root';
+import { when } from 'mobx';
+import { Checkbox } from '@patternfly/react-core';
+import PlatformFilter from '.';
```

Comment:
nit:
can you please organise all the imports in the way that all external dependencies are together at top and internal together?